service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ChildSubmissionControllerTest.java

```diff
@@ -68,18 +71,104 @@ void shouldReturnNoErrorsWhenAllDatesOfBirthAreTodayOrInPast() throws Exception
         ZonedDateTime yesterday = today.minusDays(1);
 
         AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(
-            new Children(createChild(today), createChild(yesterday))
+            new OldChildren(createChild(today), createChild(yesterday))
         );
         assertThat(callbackResponse.getErrors()).doesNotContain(ERROR_MESSAGE);
     }
 
-    private Child createChild(ZonedDateTime dateOfBirth) {
-        return new Child(null, Date.from(dateOfBirth.toInstant()), null, null, null,
+    private OldChild createChild(ZonedDateTime dateOfBirth) {
+        return new OldChild(null, Date.from(dateOfBirth.toInstant()), null, null, null,
             null, null, null, null, null, null,
-            null, null, null, null, null);
+            null, null, null, null, null, null,
+            null, null, null, null, null, null,
+            null);
+    }
+
+    @Test
+    void shouldReturnDateOfBirthErrorsForNewChildWhenFutureDateOfBirth() throws Exception {
+        CallbackRequest request = CallbackRequest.builder()
+            .caseDetails(CaseDetails.builder()
+                .id(12345L)
+                .data(ImmutableMap.of(
+                    "children1", ImmutableList.of(
+                        ImmutableMap.of(
+                            "id", "123",
+                            "value", MigratedChildren.builder()
+                                .party(ChildParty.builder()
+                                    .dateOfBirth(Date.from(ZonedDateTime.now().plusDays(1).toInstant()))
+                                    .build())
+                                .build()
+                        )
+                    )
+                ))
+                .build())
+            .build();
+
+        AboutToStartOrSubmitCallbackResponse callbackResponse = makeRequest(request);
+
+        assertThat(callbackResponse.getErrors()).contains(ERROR_MESSAGE);
+    }
+
+    @Test
+    void shouldReturnDateOfBirthErrorsForNewChildrenWhenThereIsMultipleRespondents() throws Exception {
```

Comment:
when there is multiple respondents eh? interesting

Reply:
I'll deny plagiarism to my grave Sam!